google-cloud-dataqna-v1alpha/synth.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool as s
+import synthtool.gcp as gcp
+import synthtool.languages.ruby as ruby
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICMicrogenerator()
+library = gapic.ruby_library(
+    "dataqna", "v1alpha",
+    extra_proto_files=[
+        "google/cloud/common_resources.proto",
+    ],
+    generator_args={
+        "ruby-cloud-gem-name": "google-cloud-dataqna-v1alpha",
```

Comment:
We have four existing BigQuery "family" gems. Should this one be named similarly with `bigquery` in the gem name?

* `google-cloud-bigquery-connection`
* `google-cloud-bigquery-data_transfer`
* `google-cloud-bigquery-reservation`
* `google-cloud-bigquery-storage`

Reply:
I think it would have made sense to have done so, but the team namespaced this particular API without "bigquery", and we're following suit in the clients (across languages).